playwright.config.ts

```diff
@@ -45,28 +45,28 @@ const config: PlaywrightTestConfig = {
   projects: [
     {
       name: 'chromium',
-      grepInvert: /(^(?!@desktop)$|(@tablet|@mobile))/,
+      grepInvert: /(@mobile|@tablet)(\s|$)/,
```

Comment:
The regexes didn't handle some cases (and it was too complicated to do so), so have changed these to some simpler tags (`@tablet-and-desktop` for both tablet and desktop).